### ISSUE TITLE
(Feature Request) Overview of all members with voting rights

### ISSUE THREAD

**Issue by blueforce** [joomla/volunteers.joomla.org#231]
Similar to the team listing (e.g. https://volunteers.joomla.org/teams/automated-tests-working-group) there should be a list of all members entitled to vote

It's just a new query that already exists for the secretary anyway. No additional personal data is collected for data protection. One uses only the data already existing in the user profile.

so everyone can see who is entitled to vote with one click

**Comment by sanderpotjer** [comment]
@blueforce so not really "Team with all members having voting rights" but "Overview of all members with voting rights". Correct?

**Comment by blueforce** [comment]
yes...

**Comment by jeckodevelopment** [comment]
> It's just a new query that already exists for the secretary anyway. No additional personal data is collected for data protection. One uses only the data already existing in the user profile.

Currently I've different filters in the backend to see this extraction.

**Comment by blueforce** [comment]
Yes, I suspected that the query already exists. Now it would be helpful if we could see the list publicly.

**Comment by blueforce** [comment]
Any news about this?
